regression/cbmc/address_space_size_limit3/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--32 --little-endian --object-bits 25 --pointer-check
```

Comment:
Please comment either in this test,desc or in the test body what this assembly is testing